Resources/Prototypes/_Sunrise/Catalog/ert_uplink_catalog.yml

```
@@ -811,6 +917,18 @@
   - !type:ListingLimitedStockCondition
     stock: 6
 
+- type: listing
+  id: NTUplinkSunriseTruncheon
+  productEntity: Truncheon
+  icon: { sprite: _Sunrise/Interface/Misc/nt_icons.rsi, state: truncheon }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)